neural_compressor/compression/pruner/pruners.py

```diff
@@ -73,6 +77,10 @@ def get_pruner(config, modules):
     Raises: AssertionError: Cuurently only support pruners that have been registered in PRUNERS.
     """
     ## do the ugly work here
+    ## check if it is doing self-multihead-attention pruning
+    if "mha" in config["pattern"]:
+        return PRUNERS["mha"](config, modules)
```

Comment:
block_mask pruner could not apply on mha pattern currently, please add a check here